Signal/translations/en.lproj/Localizable.strings

```diff
@@ -481,6 +481,9 @@
 /* No comment provided by engineer. */
 "NETWORK_STATUS_TEXT" = "You can check your network status by looking at the colored bar above your inbox.";
 
+/* Text for button to start a new conversation with a non-contact */
+"NEW_CONVERSATION_FOR_NON_CONTACT_FORMAT" = "New conversation with %@";
```

Comment:
> I also want to talk through the UX when typing in the wrong number,

Yes, that's just one problem. Another problematic scenario is where the current user has no signal contacts.  This whole UI changes in that case, and this fix doesn't affect that UI.  There's a ton of stuff I'd like to change here, and I'm trying to constrain this PR.